orttraining/orttraining/training_api/utils.h

```diff
@@ -10,6 +10,28 @@
 namespace onnxruntime {
 namespace training {
 namespace api {
+
+struct ModelIdentifiers {
+  const std::string train_model;
```

Comment:
Shall we make this also optional instead of using an empty string to represent loading from a buffer?